roles/openshift_logging/tasks/main.yaml

```diff
@@ -12,9 +12,10 @@
 
 - debug: msg="Created temp dir {{mktemp.stdout}}"
 
-- name: Ensuring ruamel.yaml package is on target
-  command: yum install -y ruamel.yaml
-  check_mode: no
+- name: Ensuring python2-ruamel-yaml package is on target
+  package:
```

Comment:
Is the purpose of this task to provide the dependency for the module? If so, it seems like we should add this task to lib_openshift or perhaps find another way to ensure the package is present rather than requiring all roles that use the modules to ensure the package is installed explicitly.